sci-wms/meta.yaml

```diff
@@ -22,7 +22,7 @@ requirements:
         - pytz
         - djangorestframework
         - pyugrid
-        #- pysgrid  # We need a release!
+        - pysgrid == 0.1.0
```

Comment:
@ayan-usgs conda's syntax for pinning version is different from pip's (it sucks, I know...), but I prefer if we do not pin the version.